examples/graphene_apps/python_worker/avalon_worker/tests/test_worker_encryption.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import logging
+import base64
+import random
+from hashlib import sha256
+import avalon_worker.worker_encryption as worker_encryption
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+def main(argv):
+
+    encrypt = worker_encryption.WorkerEncrypt()
+
+    # Test session key Encryption and Decryption
+    encrypt.generate_rsa_key()
+    worker_rsa_public_key = encrypt.get_rsa_public_key()
+    logger.info("worker_rsa_public_key = {}".format(worker_rsa_public_key))
+
+    session_key = encrypt.generate_session_key()
+    encrypted_session_key = encrypt.encrypt_session_key(session_key)
+    decrypted_session_key = encrypt.decrypt_session_key(encrypted_session_key)
+    if session_key == decrypted_session_key:
+        logger.info("Session key Encryption and Decryption Success")
+    else:
+        logger.error("Session key Encryption and Decryption Success")
+
+    # Test Encryption and Decryption
+    message = "Hello world"
+    session_key = encrypt.generate_session_key()
+    session_key_iv = encrypt.generate_iv()
+    encrypt_msg_bytes = encrypt.encrypt_data(
+        message.encode("utf-8"), session_key, session_key_iv)
+    decrypt_message = encrypt.decrypt_data(
+        encrypt_msg_bytes, session_key, session_key_iv)
+    decrypt_message = decrypt_message.decode("utf-8")
+
+    if message == decrypt_message:
+        logger.info("Message Encryption and Decryption Success")
+    else:
+        logger.error("Message Encryption and Decryption Success")
+
+    # Test inData Encryption and Decryption
+    input_json = dict()
```

Comment:
Use sdk function workOrderParams to create work order request.

Reply:
same as https://github.com/hyperledger/avalon/pull/612/files#r451024639